### ISSUE TITLE
CSS changes per accessibility review

### ISSUE THREAD

**Issue by kaylaondracek** [mlibrary/fishrappr#194]
## Description: 
CSS changes per Colin's accessibility review.

Accessibility review for reference:
https://docs.google.com/document/d/15WNvjeG9KFSl0PIqLsASlAyNnGIHFwHfriLzs052nWQ/edit

## Acceptance Criteria:
General 
- [ ] Add min-width to all pages
- [x] Ensure text size is at least 14px throughout
- [ ] ~~Em's need to be be used instead of px for the font-size property.~~
- [ ] define focus style for links, form elements, (others?). Default style in OSX isn't high enough contrast for some elements.

Search results
- [ ] Remove fallback color for highlighted text

Image viewer
- [ ] for highlights, reduce fill-opacity to .15 (The highlights over the text are a bit too low contrast. Either only outline the highlighted text or (with the current color) reduce the fill-opacity to 0.15 at the most.)
- [x] The focus state for the Zoom/Print buttons should still have proper contrast. Currently the text goes to black and the contrast becomes too low.

_Gordon: I moved some items back to HTML issue #175_

**Comment by gordonleacock** [comment]
This is from the last review. I think we should open a new ticket for the results of the pending review.
